### DISCUSSION TITLE
Is it possible for a task to trigger the assistant?

### DISCUSSION THREAD

**Discussion by tomkrush** [zed-industries/zed#16594] [Feature Requests]
I want to create a code review task that can run a diff between commits and uses the AI assistant to help provide feedback. I experimented with the new /terminal command but that requires multiple steps to work.

**Comment by alexanderjeurissen** [comment]
I was looking into this as well, but with a different aim.

I'd like to run a diff between a feature branch, and our main branch and use the assistant to describe the overall changeset in a condensed summary.

I think currently the best you can do is indeed spawn a task that creates a new terminal, then have a prompt in your prompt library that looks at the full terminal output.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
